src/Components/Web.JS/src/Boot.WebAssembly.ts

```diff
@@ -32,13 +32,5 @@ Blazor.start = boot;
 window['DotNet'] = DotNet;
 
 if (shouldAutoStart()) {
-  boot().catch(error => {
-    if (typeof Module !== 'undefined' && Module.err) {
-      // Logs it, and causes the error UI to appear
-      Module.err(error);
-    } else {
-      // The error must have happened so early we didn't yet set up the error UI, so just log to console
-      console.error(error);
-    }
-  });
+  boot().catch(printErr);
```

Comment:
This change adds showing the error UI element, which seems logical IMO

Reply:
Original code did the same via `Module.err` which is `printErr`